src/router/routes.js

```diff
@@ -7,10 +7,28 @@ export const routes = [
     path: '/about',
     component: () => import('~/pages/about.vue'),
   },
-    {
+  {
     path: '/contact-us',
     component: () => import('~/pages/contact-us.vue'),
   },
+  {
+    path: '/account',
```

Comment:
Later we'll implement some advanced Routing so we can always return to the previous route, once in a purchase process. So we might come back here in this structure.